### PR TITLE
XZ updates for linux arm support

### PR DESCRIPTION
**Pull request by Attempt3035**
@thesayyn I've moved your work to a new module revision as we don't want to modify already released versions on the BCR. Your work looks great, I also added linux arm to the ci/cd testing so it should pass that too now, and we will know for the future if something breaks linux arm support! Feel free to PR into my branch if any further changes are needed so they come through this PR, or I can add you to my BCR repo fork if needed👍

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @Vertexwahn, modules you maintain (xz) have been updated in this PR. Please review the changes.

File: modules/xz/metadata.json [hunk 1]
```diff
@@ -16,7 +16,8 @@
         "github:tukaani-project/xz"
     ],
     "versions": [
-        "5.4.5"
+        "5.4.5",
+        "5.4.5.bcr.1"
```
Comment (thesayyn) [inline modules/xz/metadata.json:20]: hmm, i am not sure how i feel about `debian style version` strings. Usually the version of a ruleset, or module in this case, shouldn't be directly tied to the underlying software. But this isn't a ruleset so idk...

**Comment by thesayyn** [comment]
Looks great. @dzbarsky PTAL

File: modules/xz/metadata.json (hunk 1 above)
Comment (Attempt3035) [reply modules/xz/metadata.json:20]: Not sure, I went with the method in BCR docs which seems to be how most other modules do it, version matches the software and revisions after .bcr, @fmeum thoughts on versioning like this?

File: modules/xz/metadata.json [hunk 2]
```diff
@@ -16,7 +16,8 @@
...
     ],
     "yanked_versions": {}
```
Comment (DavidZbarsky-at) [inline modules/xz/metadata.json:22]: should we yank the previous one?

**Review by DavidZbarsky-at — APPROVED** [review]
nice, lgtm!

File: modules/xz/metadata.json (hunk 1 above)
Comment (fmeum) [reply modules/xz/metadata.json:20]: This looks good to me in this situation: We are tracking an upstream that isn't Bazel-aware, so the Debian style versioning makes sense.

File: modules/xz/metadata.json (hunk 2 above)
Comment (fmeum) [reply modules/xz/metadata.json:22]: Let's do that as a follow-up so that we can recommend the fixed version.

**Review by fmeum — APPROVED** [review]

**Comment by Attempt3035** [comment]
Any ideas why the patching failed on this? I'll retry with a differently formatted patch file

**Comment by Attempt3035** [comment]
@thesayyn @DavidZbarsky-at Provided this is working on your linux arm machines, I'm thinking this is just a CI/CD pipeline issue and we should remove the linux arm tests for now, so we can push this to BCR. Thoughts?

**Comment by dzbarsky** [comment]
@Attempt3035 yeah that sounds good. The currently published module is clearly broken, so this can't get any worse :)

**Comment by bazel-io** [comment]
Hello @Vertexwahn, modules you maintain (xz) have been updated in this PR. Please review the changes.

**Review by Vertexwahn — APPROVED** [review]

**Review by bazel-io — APPROVED** [review]
Hello @bazelbuild/bcr-maintainers, all modules in this PR have been approved by their maintainers. Please take a final look to merge this PR.

**Pull request merged**